web/src/tippyjs.js

```diff
@@ -188,6 +188,20 @@ export function initialize() {
         },
     });
 
+    delegate("body", {
+        target: ".subsection-changes-save",
+        appendTo: () => document.body,
+        onShow(instance) {
+            const $elem = $(instance.reference);
+            if ($($elem).find("#save-discard-widget-id").is(":disabled")) {
+                const content = $t({defaultMessage: "Cannot save invalid Jitsi server URL"});
```

Comment:
Since the issue description asks for a period `.` at the end of the line, let's include that.